dll/win32/shell32/CDefView.cpp

```diff
@@ -54,6 +54,20 @@ typedef struct
 // to call TrackPopupMenu and let it use the 0 value as an indication that the menu was canceled
 #define CONTEXT_MENU_BASE_ID 1
 
+static UINT
+GetContextMenuFlags(IShellBrowser *pSB, SFGAOF sfgao)
+{
+    UINT cmf = CMF_NORMAL;
+    if (GetKeyState(VK_SHIFT) < 0)
+        cmf |= CMF_EXTENDEDVERBS;
+    if (sfgao & SFGAO_CANRENAME)
+        cmf |= CMF_CANRENAME;
+    HWND hwnd = 0;
```

Comment:
```suggestion
    HWND hwnd = NULL;
```

Reply:
(re: "style says NULL and not zero": The underlying reason is because a handle is like a pointer.)